app/views/home/index.html.haml

```diff
@@ -57,20 +57,28 @@
           %img{:src => asset_path("check-outlined.svg")}/
           %p
             = benefit
-
-    %div.home-upload-ontology-button
-      = upload_ontology_button
+    .home-ontologies-actions
+      .home-upload-ontology-button
+        = upload_ontology_button
+      .discover-ontologies
+        = render Buttons::RegularButtonComponent.new(id: 'discover-ontologies-button', value: t('home.discover_ontologies_button'), variant: "secondary", state: "regular", href: "/ontologies") do |btn|
```

Comment:
add a helper for this, as you did for `upload_ontology_button`, otherwise, it is some how hard to read